doc/flatpak-config.xml

```diff
@@ -62,7 +62,17 @@
                    The languages that are included when installing Locale extensions.
                    The value is a semicolon-separated list of two-letter language codes,
                    or one of the special values * or all. If this key is unset, flatpak
-                   defaults to including just the current locale.
+                   defaults to including the default-languages key and the current locale.
+                </para></listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><varname>default-languages</varname></term>
+                <listitem><para>
+                   This key is used when languages is not set, and it defines locale
+                   extensions on top of the system configured languages. The value is a
+                   semicolon-separated list of two-letter language codes, or one of the
+                   special values * or all. If this key is unset, this defaults to system
+                   configured languages.
```

Comment:
We should not support (or document) the `*`/`all` feature for this, it makes no sense for something that is additive, nor will it really work (as all means making this empty, which will merge with the system one causing it to not be empty anymore). If you need that override the full `languages` keys.